### DISCUSSION TITLE
Installing omni.kit.asset_converter?

### DISCUSSION THREAD

**Discussion by danieltmeta** [isaac-sim/IsaacLab#1421] [Q&A]
Is there a way to install the omni.ki.asset_converter library without using the omniverse GUI installation? I am trying to use a python script to access the module. A good place to start is trying to run the import without the ModuleNotFound error. Thanks!

https://docs.omniverse.nvidia.com/extensions/latest/ext_asset-converter.html

**Comment by RandomOakForest** [comment]
Currently, you will need the [Extension Manager](https://docs.omniverse.nvidia.com/extensions/latest/ext_core/ext_extension-manager.html#) to find and enable extensions. Additionally, you can select the button of "autoload" at the top of the panel of the extension of interest so that extensions are loaded upon startup. [These tutorials on Omniverse programming may be of help](https://docs.omniverse.nvidia.com/extensions/latest/ext_programming.html).

**Comment by liujin1992112** [comment]
Is there a way to install the omni.ki.asset_converter library without using the omniverse GUI installation? I am trying to use a python script to access the module. A good place to start is trying to run the import without the ModuleNotFound error. Thanks!  Me too
